tools/docker/builder/openwrt/profiles_feeds/netgear-rax40.yml

```diff
@@ -0,0 +1,30 @@
+  - name: feed_target_mips
+    uri: https://intel.prpl.dev/prplmesh/feed_target_mips.git
+    hash: 78b0b2e9675ced7dd57829da7a27592dbb483fd3
+  - name: feed_datapath
+    uri: https://intel.prpl.dev/intel/feed_datapath.git
+    hash: 23afe7e659ed6717224659aef74af2bb11d8091a
+  - name: feed_ppa
+    uri: https://intel.prpl.dev/intel/feed_ppa.git
+    hash: 4f49fa34bbd51c7dd218df164e32c17694bed421
+  - name: feed_gphy_firmware
+    uri: https://intel.prpl.dev/intel/feed_gphy_firmware.git
+    hash: cbf75f2abde587ac2c5462d117b98e3dadaae5b6
+  - name: feed_switch_api
+    uri: https://intel.prpl.dev/intel/feed_switch_api.git
+    hash: af078af5408654ca2e759ea7afcf92163282e86e
+  - name: feed_wlan_6x
+    uri: https://intel.prpl.dev/prplmesh/feed_wlan_6x.git
+    hash: 8b7ff7d902f35b41864fbf306d45ad4ed91eaa89
+  - name: feed_prpl
+    uri: https://git.prpl.dev/prplmesh/feed-prpl.git
+    hash: 6453bb7a20eff2efb031ae4aecbc51a329d68957
+  - name: feed_opensource_apps
+    uri: https://intel.prpl.dev/intel/feed_opensource_apps.git
+    hash: 4ab5c90e08ce40b5208e6aea3e594a275f8940e9
+  - name: feed_pending
+    uri: https://intel.prpl.dev/intel/feed-pending.git
+    hash: 352365b31f968ab37e772b23c30ea32dbff9e337
+  - name: feed_bridge_utils
+    hash: 35de289cc67706f1016fa8f34512eb4c337bca47
+    uri: https://git.prpl.dev/prplmesh/feed-intel-bridge-utils.git
```

Comment:
(nitpick) all the other ones have uri first, then hash (which makes more sense IMO).

Also, if intel-bridge-utils was dropped upstream, maybe we don't need it any more?